site/design/displaying-data/index.jsx

```diff
@@ -41,6 +41,17 @@ export default (
         </figure>
       </div>
 
+      {/* Tree Grid */}
+      <div className={pf('site-side-by-side grid wrap p-vertical--x-large')}>
```

Comment:
It is possible but we are considering to remove this feature from our code base and might recommend using PostCSS instead.